git-remote-gosh/src/blockchain/commit/save.rs

```diff
@@ -24,76 +24,80 @@ pub struct DeployCommitParams {
     upgrade: bool,
 }
 
-#[instrument(level = "debug")]
-pub async fn push_commit(
-    context: &mut GitHelper<impl BlockchainService>,
-    commit_id: &ObjectId,
-    branch: &str,
-) -> anyhow::Result<()> {
-    let mut buffer: Vec<u8> = Vec::new();
-    let commit = context
-        .local_repository()
-        .objects
-        .try_find(commit_id, &mut buffer)?
-        .expect("Commit should exists");
+#[async_trait]
+pub trait BlockchainCommit: BlockchainService + BlockchainUserWallet {
```

Comment:
why?
- How is that BlockchainCommit if it accepts commits?
- Why it needs Blockchain Service and BlockchainUserWallet?